core/src/components/modal/readme.md

```diff
@@ -917,6 +1074,11 @@ export class ModalExample {
         <ion-modal isOpen={true}>
           <app-stencil-component title="Ionic"></app-stencil-component>
         </ion-modal>
+        
+        {/* Require Action Sheet confirmation before dismissing */}
```

Comment:
Similar question regarding shorthand syntax.

Reply:
What's the question?

Reply:
I didn't click on the correct line when reviewing, clearly 😆  on line 1079 I believe, using the shorthand syntax for property binding:
```
canDismiss={canDismiss}
```

Reply:
Oh okay yeah -- I've updated that 👍